Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -699,8 +706,12 @@ private bool HandleActivateItemInWorld(ICommonSession? session, EntityCoordinate
 
             var activateMsg = new ActivateInWorldEvent(user, used);
             RaiseLocalEvent(used, activateMsg, true);
-            if (!activateMsg.Handled)
-                return false;
+            if (activateMsg.Handled)
+            {
+                _useDelay.BeginDelay(used, delayComponent);
+                _adminLogger.Add(LogType.InteractActivate, LogImpact.Low, $"{ToPrettyString(user):user} activated {ToPrettyString(used):used}");
+                return true;
+            }
 
             _useDelay.BeginDelay(used, delayComponent);
             _adminLogger.Add(LogType.InteractActivate, LogImpact.Low, $"{ToPrettyString(user):user} activated {ToPrettyString(used):used}");
```

Comment:
This seems like a big chance that probably shouldn't be here? If the event isn't handled it shouldn't be doing the interaction and I think this should be reverted.